system/Makefile.am

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2015 Alon Bar-Lev <alon.barlev@gmail.com>
+#
+MAINTAINERCLEANFILES= $(srcdir)/Makefile.in
+CLEANFILES = \
+	netdata.service \
+	$(NULL)
+
+include $(top_srcdir)/build/subst.inc
+
+SUFFIXES = .in
+
+dist_noinst_DATA = \
```

Comment:
there is no major difference between EXTRA_DIST and dist_noinst_DATA, I prefer to use the same method for both, please notice that both are not installed.